### ISSUE TITLE
Improve error message

### ISSUE THREAD

**Issue by djeang** [jerkar/protobuf-plugin#3]
When no input file found. It would be nice to show where are supposed poto file to be found. as Missing input file : expected location src/main/proto.  This should be done simply by toString on the JkFileTree.

```
Method : generateSources
------------------------
Starting program : [protoc, --java_out=C:\Users\djeang\eclipse-workspaces\jerkar\toto\build\output\proto] ... 
|  Missing input file.
|  Method generateSources failed in 0.015 seconds.
```

**Comment by cuchaz** [comment]
Sadly, I don't any control over that error message since it's written by `protoc`.

If we show an error in Java when the source file list is empty (which is what I think you're suggesting), that would remove the ability for users to have non-source-file-based invocations of `protoc`, like:
```
JkProtobuf.of(this).andExtraArgs("--version").compile();
```
This is a trivial example of course, but `protoc` has different use cases where source files are not used. I'd hate to restrict the usefulness of `andExtraArgs()` by throwing errors unnecessarily.

**Comment by djeang** [comment]
Ok may we can just remind the user about the actual attributes of current  JkProtobuf instance prior compiling `compiling protobuf with arguments : protobuf file locations = ......, other argument = ...`. As such it is easier to figure out what happens.

**Comment by cuchaz** [comment]
how about this? 74ae81e0e68887f773165e76efbbfa1903b6b276

**Comment by cuchaz** [comment]
fixed by #6
